llvm/lib/Transforms/InstCombine/InstCombineInternal.h

```diff
@@ -634,6 +634,11 @@ class LLVM_LIBRARY_VISIBILITY InstCombinerImpl final
   Instruction *foldPHIArgZextsIntoPHI(PHINode &PN);
   Instruction *foldPHIArgIntToPtrToPHI(PHINode &PN);
 
+  // If the phi is within a phi web, which is formed by the def-use chain
```

Comment:
```suggestion
  /// If the phi is within a phi web, which is formed by the def-use chain
```
Please use `///` for header comments: https://llvm.org/docs/CodingStandards.html#comment-formatting.